src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -98,7 +98,7 @@ public bool MapInboundClaims
             }
             set
             {
-                if(!_mapInboundClaims && value && _inboundClaimTypeMap.Count == 0)
+                if (!_mapInboundClaims && value && _inboundClaimTypeMap.Count == 0)
```

Comment:
try not to make these spacing changes, it is just noise and may make integration into dev harder.

Reply:
Must have been something my editor did; I did not manually change any spacing. I'll clean it up.